modules/lib/lib-content/src/main/java/com/enonic/xp/lib/content/ResetInheritanceHandler.java

```diff
@@ -0,0 +1,83 @@
+package com.enonic.xp.lib.content;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import com.google.common.base.Preconditions;
+
+import com.enonic.xp.content.Content;
+import com.enonic.xp.content.ContentId;
+import com.enonic.xp.content.ContentInheritType;
+import com.enonic.xp.content.ContentPath;
+import com.enonic.xp.content.ResetContentInheritParams;
+import com.enonic.xp.content.SyncContentService;
+import com.enonic.xp.project.ProjectName;
+import com.enonic.xp.script.bean.BeanContext;
+
+public class ResetInheritanceHandler
+    extends BaseContextHandler
+{
+    private SyncContentService syncContentService;
+
+    private String key;
+
+    private String projectName;
+
+    private List<String> inherit;
+
+    @Override
+    public void initialize( final BeanContext context )
+    {
+        super.initialize( context );
+        this.syncContentService = context.getService( SyncContentService.class ).get();
```

Comment:
call `get()` in `doExecute` 
Due to OSGi dynamic nature service may become unavailable between `initialize` and `doExecute`.